Mvx.Geocoder.Sample.Core/App.cs

```diff
@@ -1,18 +1,18 @@
-using MvvmCross.Core.ViewModels;
-using MvvmCross.Platform.IoC;
+﻿using MvvmCross.ViewModels;
+using Mvx.Geocoder.Sample.Core.ViewModels;
 
 namespace Mvx.Geocoder.Sample.Core
 {
-    public class App : MvxApplication
-    {
-        public override void Initialize()
-        {
-            CreatableTypes()
-                .EndingWith("Service")
-                .AsInterfaces()
-                .RegisterAsLazySingleton();
-				
-            RegisterAppStart<ViewModels.FirstViewModel>();
-        }
-    }
-}
\ No newline at end of file
+	public class App : MvxApplication
+	{
+		public override void Initialize()
+		{
+			//CreatableTypes()
+			//	.EndingWith("Service")
+			//	.AsInterfaces()
+			//	.RegisterAsLazySingleton();
+
+			RegisterAppStart<FirstViewModel>();
+		}
+	}
+}

```

Comment:
It is important to keep the author's code formatting (in that case spaces should not be changed to the tabs).

(no need to fix that in that case, but please be careful in other PRs)